src/ArticleQualityService.types.ts

```diff
@@ -43,9 +43,7 @@ export interface Result extends WikidataRevision {
 }
 
 export interface OresScoresResponse {
```

Comment:
While we are at it, let's change the name of this interface to something more generic like `QualityScoresResponse`. We are no longer relying on Ores. The same applies to the interfaces below.